uuid.go

```diff
@@ -47,18 +47,16 @@ var (
 	poolMu      sync.Mutex
 	poolPos     = randPoolSize     // protected with poolMu
 	pool        [randPoolSize]byte // protected with poolMu
-)
-
-type invalidLengthError struct{ len int }
 
-func (err invalidLengthError) Error() string {
-	return fmt.Sprintf("invalid UUID length: %d", err.len)
-}
+	ErrInvalidUUIDFormat      = errors.New("invalid UUID format")
+	ErrInvalidURNPrefix       = errors.New("invalid urn prefix")
```

Comment:
This should be a new type:
```
type URNPrefixError struct{ Prefix string }
```

Reply:
Solved in [54f9572](https://github.com/google/uuid/pull/166/commits/54f95728c20a700436b4f3a8e699a99235748175)